src/components/DocumentationTopic/TopicsTable.vue

```diff
@@ -110,8 +110,20 @@ export default {
     sectionsWithTopics() {
       return this.sections.map(section => ({
         ...section,
-        topics: section.identifiers.reduce(
-          (list, id) => (this.references[id] ? list.concat(this.references[id]) : list),
+        topics: section.identifierItems.reduce(
+          (list, item) => {
+            if (this.references[item.identifier]) {
+              const ref = this.references[item.identifier];
+              if (item.overrideTitle) {
+                ref.title = item.overrideTitle;
+              }
+              if (item.overridingTitleInlineContent) {
```

Comment:
I think the reason that this might not be working like you expect is that the logic for `overridingTitle` and `overridingTitleInlineContent` lives in `ContentNode`, which is not used by the `TopicsLinkBlock` component that renders the see-also links (it directly uses `<a>` and `<router-link>` components).

Reply:
I think Marcus is right. It would probably use the `overrideTitle` as `TopicsLinkBlock` already uses the `title` property, but it does not know what `titleInlineContent` is.

Since that property is a RenderJSON format, we would have to conditionally render the `ContentNode` inside the `TopicsLinkBlock`, only when `titleInlineContent` is present for example. Similar to how we call `renderChildren(titleInlineContent)` in `ContentNode.vue`.

I am not sure how we should deal with the `titleTag` in that situation either.